### PR TITLE
Add display_name modification in insights_register.py

### PR DESCRIPTION
**Pull request by kubealex**
As of now, the variable is never uses, making it useless.
I added a check and edited the command accordingly.

### CODE REVIEW

File: plugins/modules/insights_register.py [hunk 1]
```diff
@@ -144,7 +144,10 @@ def run_module():
             result['message'] = 'New machine-id created - ' + insights_name + ' has been registered'
             module.exit_json(**result)
         else:
-            subprocess.call([insights_name, '--register'])
+            if display_name:
+                subprocess.call([insights_name, '--register', '--display-name=' + display_name])
+            else:
+                subprocess.call([insights_name, '--register'])
```
Comment (ptoscano) [inline plugins/modules/insights_register.py:150]: can you please avoid the duplication by factorizing the command line arguments? something like:
```python
args = [...]
if something_else:
    args.append(...)/.extend(...)
subprocess.call(args)
```

**Review by ptoscano — CHANGES_REQUESTED** [review]
Thanks @kubealex for the contribution!

First of all: the CI jobs were recently fixed, so please rebase this on top of the current `master` branch.

I left a couple of comments for things to fix (nothing complicated).

Also, should `--display-name` be specified also when forcefully registering, i.e. the system is already registered and `--force-reregister` is passed?

**Comment by kubealex** [comment]
Thank you @ptoscano! I added the modifications based on your suggestion for all the commands.

**Review by ptoscano — CHANGES_REQUESTED** [review]
Note there are few style issues:
```
ERROR: Found 6 pep8 issue(s) which need to be resolved:
ERROR: plugins/modules/insights_register.py:135:1: W293: blank line contains whitespace (100%)
ERROR: plugins/modules/insights_register.py:145:71: W291: trailing whitespace (100%)
ERROR: plugins/modules/insights_register.py:151:49: W291: trailing whitespace (100%)
ERROR: plugins/modules/insights_register.py:153:71: W291: trailing whitespace (100%)
ERROR: plugins/modules/insights_register.py:166:51: W291: trailing whitespace (100%)
ERROR: plugins/modules/insights_register.py:167:43: W291: trailing whitespace (100%)
```
Can you please squash everything in a single commit?

**Comment by kubealex** [comment]
Yep, sorry Pino, I will fix the error and squash the commits asap. Thank you!

**Pull request closed without merging**